src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts

```diff
@@ -70,7 +70,7 @@ export class CreateClusterReviewComponent implements OnInit {
 
       _.forEach(this.services, (serviceKey) => {
         this.serviceOccurrences[serviceKey['service_type']] =
-          (this.serviceOccurrences[serviceKey['service_type']] || 0) + 1;
+          this.serviceOccurrences[serviceKey['service_type']] + 1 || 1;
```

Comment:
This won't fix the issue for count, you need to subscribe to host list and fetch services from there instead (as it includes both ceph_hosts and orch_hosts)